images/router/haproxy/conf/haproxy-config.template

```diff
@@ -459,7 +465,11 @@ backend be_secure_{{$cfgIdx}}
   balance {{ $balanceAlgo }}
       {{ end }}
     {{ else }}
+      {{ if (matchPattern "roundrobin|leastconn|source" (env "ROUTER_LOAD_BALANCE_ALGORITHM" "")) }}
+  balance {{ env "ROUTER_LOAD_BALANCE_ALGORITHM" "leastconn"}}
```

Comment:
Should this environment variable be added to the DC as well? It would make it easier for the user and make the doc easier as well.